flit_scm is not available on Solaris, but in this case, flit_core
can be used as well with no changes to deliverables.

Not intended for upstream.

--- exceptiongroup-1.1.1/pyproject.toml
+++ exceptiongroup-1.1.1/pyproject.toml
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["flit_scm"]
-build-backend = "flit_scm:buildapi"
+requires = ["flit_core"]
+build-backend = "flit_core.buildapi"
 
 [project]
 name = "exceptiongroup"
